### PR TITLE
[WebBundle] Finished the frontend account/profile view

### PR DESCRIPTION
**Pull request by pix-art**
I needed this for my own project so I implemented this for the whole sylius project. Sharing is caring

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
@pix-art Cool, could you give us a quick screenshot? :)

File: src/Sylius/Bundle/WebBundle/Menu/FrontendMenuBuilder.php [hunk 1]
```diff
@@ -181,21 +181,27 @@ public function createMainMenu()
     }
 
     /**
-     * Builds frontend currency menu.
+     * Builds frontend currency menu if we have more then 1 currency to display.
```
Comment (pborreli) [inline src/Sylius/Bundle/WebBundle/Menu/FrontendMenuBuilder.php]: typo : then -> than

File: src/Sylius/Bundle/WebBundle/Menu/LocaleMenuBuilder.php [hunk 2]
```diff
@@ -55,27 +55,29 @@ public function __construct(
     }
 
     /**
-     * Builds frontend locale menu.
+     * Builds frontend locale menu if more then one language is defined.
```
Comment (pborreli) [inline src/Sylius/Bundle/WebBundle/Menu/LocaleMenuBuilder.php]: same typo : then -> than

**Comment by pix-art** [comment]
screenshot 2015-02-19 11 14 21
screenshot 2015-02-19 11 14 26
screenshot 2015-02-19 11 15 41

File: src/Sylius/Bundle/WebBundle/Menu/LocaleMenuBuilder.php (hunk 2 above)
Comment (pix-art) [inline src/Sylius/Bundle/WebBundle/Menu/LocaleMenuBuilder.php]: Fixed now :)

**Comment by pix-art** [comment]
As you can see i've also added your last 5 orders to your account home view. Because our client requested this and I felt this was pretty default behavior for a good ecommerce platform.

**Comment by pix-art** [comment]
Tests failed, working on this now

File: src/Sylius/Bundle/WebBundle/Menu/FrontendMenuBuilder.php [hunk 3]
```diff
@@ -181,21 +181,27 @@ public function createMainMenu()
...
 
     /**
-     * Builds frontend currency menu.
+     * Builds frontend currency menu if we have more than 1 currency to display.
      *
      * @return ItemInterface
      */
     public function createCurrencyMenu()
     {
+        $currencies = $this->currencyProvider->getAvailableCurrencies();
```
Comment (stloyd) [inline src/Sylius/Bundle/WebBundle/Menu/FrontendMenuBuilder.php]: Those changes should be reverted as they not related to this PR.

Comment (pix-art) [inline src/Sylius/Bundle/WebBundle/Menu/FrontendMenuBuilder.php]: Done

**Comment by kurakin-oleksandr** [comment]
:+1:

**Comment by pjedrzejewski** [comment]
Thank you Joeri! :+1:

**Comment by aramalipoor** [comment]
@aRn0D @pix-art @pjedrzejewski There's a little bug at [_list.twig.html Line 25](https://github.com/Sylius/Sylius/pull/2494/files#diff-b7a165c06fbfdab3c9c0132a7eff9596R25). It expects to see **btn()** on `buttons` variables but when `_list.twig.html` in included [here at line 24](https://github.com/Sylius/Sylius/pull/2494/files#diff-6d4bc0d713343c995668066562be64ecR24) it's passing `buttons` from `SyliusResourceBundle:Macros:buttons.html.twig` which does NOT have **btn()** macro! 

This causes **/account** to show an error like this:
```
Method "btn" for object "__TwigTemplate_5e8ce4e9721210415e5813495280ae26e27d7902d1b1cea9a1c94fb06fbb11b5" does not exist in SyliusWebBundle:Frontend/Account:Order/_list.html.twig at line 28
```

**Pull request merged**